plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/internal/ERDUIMessages.properties

```
@@ -34,11 +34,13 @@ erd_editor_control_action_refresh_diagram = Refresh Diagram
 erd_editor_control_action_print_diagram = Print Diagram
 erd_editor_control_action_configuration = Configuration
 erd_preference_page_title_attribute_style=Attribute styles
+erd_preference_page_title_routing=Routing settings
 erd_preference_page_title_attributes_visibility=Attributes visibility
 erd_preference_page_title_diagram_contents=Diagram contents
 erd_preference_page_title_shows_views=Show views
 erd_preference_page_title_shows_partitions=Show partitions
 erd_preference_page_title_color_pref = Color settings
+erd_preference_page_title_routing_combo = Routing type
```

```suggestion
erd_preference_page_title_routing_combo = Routing mechanism
```

`Type` is too generic, `Algorithm` is too scientific.